src/test/rustdoc/issue-83852.rs

```diff
@@ -0,0 +1,10 @@
+#![crate_name = "foo"]
```

Comment:
Can you rename this file to something more descriptive, like `struct-arg-pattern.rs`?